src/views/partials/labels.handlebars

```diff
@@ -0,0 +1,18 @@
+<h3>Labels</h3>
+{{!-- List Labels --}}
+<form>
+    {{#each label}}
+        <input type="checkbox" checked id="card_label_{{@index}}">
+        <label for="card_label_{{@index}}"><p class="card-label-highlight" style="background-color: {{this.color}};"></p></label>
+    {{/each}}
+    <button type="submit" value="submit">Set Changes</button>
```

Comment:
i don't think setting the value does anything, but not required to fix for me to accept this code

Reply:
Ok got rid of it